### DISCUSSION TITLE
Locked out of v14 and can't use v16 classic

### DISCUSSION THREAD

**Discussion by shishkin** [blinksh/blink#1778] [Q&A] [answered]
Hi, I bought Blink in Feb 2021 for the need to work on my IPad. I since switched to a Mac and didn't pay attention to the change in pricing strategy. Just recently I noticed a new Blink icon and realized that my Blink is not usable anymore. After installing the new v16 I don't see a way to select free classic option. I don't need any build or IDE subscription, only a basic terminal. Restoring purchases didn't work.

There might be valid reasons for change in business model, but deliberately breaking a purchased app and rendering it unusable is in my opinion unethical. Possible fair alternatives would be just leaving the app without updates or refunding purchases.

Is there still a way for me to get a usable version of Blink that I purchased or should I turn to AppStore for dispute resolution?

**Comment by carloscabanero** [comment, marked as answer]
Hi @shishkin! Thanks for your message. I first want to start by responding to the "deliberately breaking a purchased app". We did not break the app deliberately and what you are experiencing is actually a result of "leaving the app without updates". Specifically, what you are seeing with rendering issues was provoked by iOS16 more than a year ago, and we started this "migration process" (forced by Apple) more than a year and half ago. The current business model change actually tries to reflect this new reality, keeping an app updated and working (specially one as complex as Blink) actually costs someone's effort and money, even to just make sure that new OSs work, whether interested in new features or not.

As mentioned above, the previous Blink was discontinued a year and a half ago, and we added the proper notices and messages inside the app and in the AppStore to direct people to the new application. Our "fair alternative" has been to grandfather users, so far tens of thousands, into a new Classic plan to make sure they would still get their updates on the functionality they purchased for the following years. Please note how "just leaving the apps without updates" as you suggested was not actually a "fair alternative", and actually putting work and creating a migration system from our side (again, this costed us money to develop and maintain for this year and half), so previous users could enjoy the classic features for the upcoming years without paying again, was actually the "Fair alternative".

Now, do not worry because even if you did not do the migration yet, we are still trying to honor it manually with Codes for as long as they last. To do so, just please reach out to us over email to support at blink.sh.

Pardon if my message sounds a bit harsh, but I'm done with accusations when we actually put our work and money trying to do the right thing for a process that has already helped tens of thousands of users. And we may have made mistakes all along for sure, but this is a complex process for a very very small team, and we are still trying to help any way we can.

**Reply by shishkin** [reply]
Thanks @carloscabanero. I didn't realize the breaking changes introduced in iOS16 made it impossible to leave the old app as is. Though I couldn't find any related issues or discussions about it here.

I understand that you've done your best to prepare the users in advance and it was my ignorance that I didn't notice it before. Changing the icon was what actually worked for me. Who reads appstore update notes anyway?

Sorry for accusations in my original post. I did search for similar issues and discussions and was surprised not to find a clear and concise answer. Now I hope this discussion will prevent future accusations :)

**Reply by carloscabanero** [reply]
All good, appreciate your message and glad we could get you on board!
